gulp/build-tasks/assets-includes.js

```diff
@@ -1,14 +1,16 @@
 'use strict';
 
+const pjson = require('../../package.json');
+
 module.exports = function (gulp, plugins, config, dest, local = false, relpath = false) {
   return function (cb) {
     let src = [
       `${config.basepath.src}/assets/_project/_blocks/layout/**/*.html`,
     ];
       //.concat(config.build.excludes); //remove concat excludes, remove from gulp-config.json also
     let cdnLink = {
-      regex: new RegExp(`="/assets/${config.versionName}`, 'g'),
-      replacement: `="https://static.qgov.net.au/assets/${config.versionName}`,
+      regex: new RegExp(`="/assets/${config.versionName}/latest`, 'g'),
```

Comment:
ok, but we also want to keep the original folder with /latest.

and make a new folder with our pjson.version